docs/rules/component-options-name-casing.md

```diff
@@ -0,0 +1,137 @@
+---
+pageClass: rule-details
+sidebarDepth: 0
+title: vue/component-options-name-casing
+description: enforce the casing of component name in `components` options
+---
+# vue/component-options-name-casing
+
+> enforce the casing of component name in `components` options
+
+- :exclamation: <badge text="This rule has not been released yet." vertical="middle" type="error"> ***This rule has not been released yet.*** </badge>
+- :wrench: The `--fix` option on the [command line](https://eslint.org/docs/user-guide/command-line-interface#fixing-problems) can automatically fix some of the problems reported by this rule.
+
+## :book: Rule Details
+
+This rule aims to enforce casing of the component names in `components` options.
+
+## :wrench: Options
+
+```json
+{
+  "vue/component-options-name-casing": ["error", "PascalCase" | "kebab-case" | "camelCase"]
+}
+```
+
+This rule has an option which can be one of these values:
+
+- `"PascalCase"` (default) ... enforce component names to pascal case.
+- `"kebab-case"` ... enforce component names to kebab case.
+- `"camelCase"` ... enforce component names to camel case.
+
+### `"PascalCase"` (default)
+
+<eslint-code-block fix :rules="{'vue/component-definition-name-casing': ['error']}">
```

Comment:
The rule name in this code block and the following ones is wrong. That's why the interactive examples in the [preview deployment](https://deploy-preview-1725--eslint-plugin-vue.netlify.app/rules/component-options-name-casing.html) don't work.

Reply:
Oh, that's a copy-paste mistake.